auth-web/src/components/auth/staff/review-task/QsApplication.vue

```diff
@@ -236,22 +273,58 @@ export default defineComponent({
       }),
       qsRequirements: computed((): QualifiedSupplierRequirementsConfig[] => {
         return userAccessRequirements[props.taskDetails?.type]
-      })
+      }),
+      isLawyerNotaryApplication: computed((): boolean => props.taskDetails?.type === TaskType.MHR_LAWYER_NOTARY),
```

Comment:
cool, I wonder what's the difference between this and 
`computed<boolean>(()=> props.taskDetails?.type === TaskType.MHR_LAWYER_NOTARY)`

Reply:
i'm not sure! 
It may be a case of either or. Both seem to do the trick.

Do you have a preference? I wonder if one is preferred in a more strict typing scenario 🤔

Reply:
either works I think, any feature flags or timing this needs to be released at? or can we just merge and push to prod anytime?